recipes-app/iot2050-firmware-update/files/iot2050-firmware-update

```diff
@@ -137,7 +137,9 @@ class FirmwareUpdate(object):
             mtd_base_path = "{}/mtd".format(ospi_dev_path)
 
         self.firmware.seek(0)
-        firmware_size = os.stat(self.firmware.fileno()).st_size
+        self.firmware.seek(0, os.SEEK_END)
+        firmware_size = self.firmware.tell() 
+        self.firmware.seek(0)
```

Comment:
As this is a regression fix for the previous commit, it should get folded into it.

Reply:
Squeshed.